src/test/java/org/broadinstitute/hellbender/engine/dataflow/TestPipeline.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.common.base.Strings;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+
+public class TestPipeline {
```

Comment:
One additional comment, I like that this drops in without any changes to the code, but I think the confusion of naming it exactly like google's will be more of a difficulty than giving this a different name like `GatkTestPipeline`.  People will accidentally import the wrong one if they're named the same I suspect.